Sources/MapboxMaps/Foundation/Extensions/Bundle+MapboxMaps.swift

```diff
@@ -10,4 +14,11 @@ extension Bundle {
         return Bundle(for: BundleLocator.self)
         #endif
     }
+
+    static var mapboxMapsMetadata: MapboxMapsMetaData = {
```

Comment:
Inconsistent capitalization of metadata (vs metaData). Looks like it's considered a single word, so let's make the d lowercase consistently